config/webpack.config.js

```diff
@@ -34,7 +37,7 @@ const commonConfig = ({ dev }) => {
         }
       : path.resolve(__dirname, '../src/index.ts'),
     output: {
-      path: path.resolve(__dirname, '../build/js'),
+      path: path.resolve(__dirname, '../build/stable/js'),
```

Comment:
This should not be needed

Reply:
I'm happy to change this and the other stable addition back, but for context I added these because I otherwise had to manually move the build output into a `/stable` directory before I was able to build an image and get it into an ephemeral env. cc @florkbr